.github/workflows/release.yml

```diff
@@ -1,8 +1,11 @@
-name: Build Releases
+name: Generate Releases
 
 on:
   release:
     types: [published]
+  push:
+    branches:
+      - 'feature/issue-57_standalone-executable'
```

Comment:
Do we update this to `master`?

Reply:
I think we delete this entirely when we're ready to merge, and create a Release instead.

Reply:
SGTM, let's keep this convo open and I'll make my final approval contingent upon deleting this.